elements/pfe-sass/extends/_typography_extends.scss

```diff
@@ -0,0 +1,39 @@
+/// EXTENDS
+/// @see {mixin} pfe-text 
+
+%pfe-content-base {
+    font-family: pfe-var(font-family);
+    line-height: pfe-var(line-height);
+    font-weight: pfe-var(font-weight--light);
+}
+
+/// List items
+%pfe-content--list {
+    padding-left: calc(pfe-var(ui--indent) * 2);
+    &:not(:last-child):not(:empty) {
+      margin-bottom: pfe-var(content-spacer--body--md); // 16px
+    }
+}
+%pfe-content--list-item {
+  &:not(:last-child):not(:empty) {
+      margin-bottom: pfe-var(content-spacer--body--sm); // 8px
+  }
+}
+%pfe-content--nested-list-item {
```

Comment:
done 👍

Reply:
Did you push up the comment?  I'm not seeing it in the diff yet.

Reply:
See my note below, I think my confusion is because this should maybe be `%pfe-content--nested-list` not `%pfe-content--nested-list-item` since `list-item` refers to the li tags and not the ol/ul tags.